src/CalcViewModel/ApplicationViewModel.cpp

```diff
@@ -204,3 +206,55 @@ void ApplicationViewModel::SetMenuCategories()
     // property setter logic.
     Categories = NavCategoryGroup::CreateMenuOptions();
 }
+
+void ApplicationViewModel::AlwaysOnTopButtonClick(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
+{
+    HandleAlwaysOnTopButtonClick(sender, e);
+}
+
+Concurrency::task<void> ApplicationViewModel::HandleAlwaysOnTopButtonClick(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
```

Comment:
Remove Concurrency::task<void> and replace with task<void> add the Concurrency namespace to the top of the file.